### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by mlfoxh** [Docile-Alligator/Infinity-For-Reddit#1516] [General]
Thank you @Docile-Alligator for building what I think is the best Reddit app out there. It's beautiful work, and I'm sorry to see its future thrown off by the site's absurd new API pricing scheme. Good luck with your future endeavors!

**Comment by ololiuhqui** [comment]
I'm gonna add a thank you too, this app was really well made and had very interesting features, sad that reddit decided to kill it...
